public/index.html

```diff
@@ -1,30 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="theme-color" content="#000000" />
+		<meta
+			name="description"
+			content="Web site created using create-react-app"
+		/>
+		<link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
 
-<head>
+		<link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css?family=Varela+Round"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/icon?family=Material+Icons"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+		/>
 
-  <meta charset="utf-8" />
-  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta name="theme-color" content="#000000" />
-  <meta name="description" content="Web site created using create-react-app" />
-  <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+		<title>Tastk Coding</title>
+		<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
```

Comment:
Are u using Jquery?
Why have you added this jquery script?
Maybe if you are using some jquery functionality you are making some bad practices

Reply:
Negrito estuve mirando, y parece que bootstraap necesita para poder funcionar, o almenos las funciones de bootstrap que estoy usando